packages/p2p/src/components/buy-sell/buy-sell-row.scss

```diff
@@ -90,6 +90,13 @@
 
     &__button {
         align-self: flex-start;
+        display: flex;
+        flex-flow: row wrap;
+        white-space: inherit;
+        height: fit-content;
+        max-height: 7rem;
+        margin-left: auto;
+        padding: 1rem 1.6rem;
```

Comment:
Added padding since height is now `fit-content`, padding is required to provide some space between the button text:

### Before:
<img width="446" alt="Screenshot 2022-07-18 at 2 33 59 PM" src="https://user-images.githubusercontent.com/103016120/179463281-b5c29962-b17a-4af5-b103-f3e4e4d6e0e5.png">

### After:
<img width="455" alt="Screenshot 2022-07-18 at 3 18 33 PM" src="https://user-images.githubusercontent.com/103016120/179463315-fe58f348-e0aa-44d2-899f-beb17e0d9ae8.png">

Reply:
Alternative way without using `padding: 1rem 1.6rem` is to unset `height: fit-content`, but this will cause the overflown button to have `0rem` top-bottom padding due to the `dc-btn` default styles of `padding: 0 1.6rem`:
<img width="446" alt="Screenshot 2022-07-18 at 3 32 36 PM" src="https://user-images.githubusercontent.com/103016120/179463859-29e0ebf1-0559-451c-a90a-129cb7368036.png">
<img width="251" alt="Screenshot 2022-07-18 at 3 33 51 PM" src="https://user-images.githubusercontent.com/103016120/179463946-b862a2fb-5b93-41ff-964c-6a8fd8e71246.png">